leapp/snactor/commands/discover.py

```diff
@@ -89,6 +90,7 @@ def _get_model_details(model):
 @command_opt('safe', is_flag=True, help='Analyze actor files statically to work around runtime errors')
 @requires_repository
 def cli(args):
+    logging.basicConfig(level=logging.WARNING, stream=sys.stderr)
```

Comment:
I wonder if it wouldn't be easier to think about putting configure_logger from leapp.logger at the end of def cli in leapp/snactor/__init__.py and remove the configure_logger calls from `snactor workflow run` and `snactor run`